File: projectBlueWater/src/main/java/com/lasthopesoftware/bluewater/shared/android/notifications/control/ControlNotifications.java
package com.lasthopesoftware.bluewater.shared.android.notifications.control;

import android.app.Notification;

public interface ControlNotifications {
	void notifyBackground(Notification notification, int notificationId);

	void notifyForeground(Notification notification, int notificationId);

	void removeAllNotifications();

	void removeNotification(int notificationId);

	void stopForegroundNotification(int notificationId);
}

File: projectBlueWater/src/main/java/com/lasthopesoftware/bluewater/shared/android/notifications/control/ControlNotifications.kt
package com.lasthopesoftware.bluewater.shared.android.notifications.control

import android.app.Notification

interface ControlNotifications {
    fun notifyEither(notification: Notification?, notificationId: Int)
    fun notifyBackground(notification: Notification?, notificationId: Int)
    fun notifyForeground(notification: Notification?, notificationId: Int)
    fun removeAllNotifications()
    fun removeNotification(notificationId: Int)
    fun stopForegroundNotification(notificationId: Int)
}
